### ISSUE TITLE
User feedback issues automatically grouped

### ISSUE THREAD

**Issue by Polydash** [getsentry/sentry-unity#1327]

**Comment by scttcper** [comment]
Moving this over to the sentry-unity project where we can hopefully get more answers

**Comment by mattjohnsonpint** [comment]
I don't believe this is an SDK issue.  Nothing has changed with how we report user feedback from the SDK side.

Transferring back to Sentry...
